pkgs/applications/audio/noisetorch/default.nix

```
@@ -0,0 +1,48 @@
+{ stdenv, cmake, buildGoModule, fetchgit }:
+
+buildGoModule rec {
+  name = "NoiseTorch-${version}";
+  version = "0.5.1-beta";
+  rev = "37b35ce1eed438adee42e50d0cc0ddf48375846e";
```

Move `rev` into `src`. Also:

```suggestion
  rev = version;
```

Since upstream tags releases.

---

done. Also moved plugin into a separate package.